pkg/collector/iptables_collector_test.go

```diff
@@ -0,0 +1,38 @@
+package collector
+
+import (
+	"testing"
+)
+
+func TestNewIPTablesCollector(t *testing.T) {
+	tests := []struct {
+		name          string
+		want          int
+		wantErr       bool
```

Comment:
question: do we want to check the error signature as well?

Reply:
❤️🙏 Thank you for asking this because this is great question for unlocking more improvement, Error handling throughout this repo is standard log.error usage, there is nothing in commonality to check. Again this is something we can discuss as thought share because I will be keen to pick this down for a dedicated “test improvement” PR.

Idea I have in mind and here is the share:

Upstream team has some way to create azure pipeline but I have no clue if they will allow to spin aks clusters if they do then we could use the whole end-to-end scenario for a pipeline. (This is all thoughts I don’t think they have any built-ins for aks cluster wipin ups and storage)

Reply:
Got it, thanks for the context!